docs/docs/legal/policies/privacy-policy.mdx

```diff
@@ -2,17 +2,36 @@
 title: Privacy Policy
 custom_edit_url: null
 --- 
-import PiidTable from '../_piid-table.mdx';
 
-Last updated on March 07, 2024
+Last updated on 20 March, 2025
 
-This privacy policy applies to CAOS Ltd., the websites it operates (including zitadel.ch, zitadel.cloud and zitadel.com) and the services and products it provides (including ZITADEL). This privacy policy describes how we process personal data for the provision of this websites and our products.
+This privacy policy describes how ZITADEL and its wholly owned subsidiaries and affiliates (collectively, "**ZITADEL**", “**CAOS**", "**we**" or "**us**") collect, use, disclose and otherwise process your personal data in connection with the management of our business and our relationships with customers, visitors and event attendees.
```

Comment:
```suggestion
This privacy policy describes how ZITADEL Inc. and its wholly owned subsidiaries and affiliates (collectively, "**ZITADEL**", “**CAOS**", "**we**" or "**us**") collect, use, disclose and otherwise process your personal data in connection with the management of our business and our relationships with customers, visitors and event attendees.
```